src/procedures/proc_functions.c

```
@@ -0,0 +1,141 @@
+
+	ctx->privateData = pdata;
+	return PROCEDURE_OK;
+}
+
+
+// promote the rax iterator to the next function and return its name and mode.
+SIValue *Proc_FunctionsStep
+(
+	ProcedureCtx *ctx
+) {
+	ASSERT(ctx->privateData);
```

cppcheck

[error] Null pointer dereference

(nullPointer)